backend/migrations/versions/79060a1bab8e_.py

```diff
@@ -18,10 +18,7 @@
 
 def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
-    op.alter_column('hacknight', 'date',
-               existing_type=sa.VARCHAR(length=20),
-               type_=sa.Date(),
-               existing_nullable=True)
+    op.execute("""ALTER TABLE hacknight ALTER COLUMN date TYPE DATE USING date::date""")
```

Comment:
And what happened here?

Reply:
@magul there was some problem with the upgrade of database with date type. It required: "USING date::date" and in SQLAlchemy it was impossible to add this "USING" part. With @stanislawK we were looking for different solutions and we found that we should use pure SQL to solve this problem.